### DISCUSSION TITLE
The website is not showing properly, only plain text. please help

### DISCUSSION THREAD

**Discussion by souvikde99** [community/community#23758] [Other Feature Feedback, Questions, & Ideas]
<p>I am reporting a problem with my website at <a href="https://souvikde99.github.io/souvik/" rel="nofollow noopener">https://souvikde99.github.io/souvik/</a></p>
<p>My web browser is XXX version XXX, running on operating system windows .10</p>
<p>(Please go to <a href="https://github.com/souvikde99/souvik" rel="nofollow noopener">https://github.com/souvikde99/souvik</a> to search for or<br>
report any problems with Biopython itself.)<br>
the website is not showing properly, only plain text. please help…<br>
but it’s working fine in offline mode. in my pc</p>

**Comment by souvikde99** [comment]
<p>How to fix it… Please help na</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/souvikde99">@souvikde99</a>, when I use the first link, all I’m getting is a 404 error. Did you take down your GitHub Pages site at that URL? I see that the repository is still there.</p>

**Reply by Nidhidahiya67764** [reply]
> Hi [@souvikde99](/souvikde99), when I use the first link, all I’m getting is a 404 error. Did you take down your GitHub Pages site at that URL? I see that the repository is still there.

Hi I'm facing the same problem my actual website looks like this in tiny host with same code https://violet-melitta-62.tiiny.site 
, however, in GitHub it just plain text https://nidhidahiya67764.github.io/
can you help me out?

**Reply by radar-blip** [reply]
@Nidhidahiya67764,

In your index.html
Try changing this: 
`<link href="/css/styles.css" rel="stylesheet" type="text/css"/>`

To this:
`<link href="styles.css" rel="stylesheet" type="text/css"/>`

Reason for this: your stylesheet (`styles.css`) is not in a folder called `css`

**Comment by souvikde99** [comment, marked as answer]
<p>Thanks for your reply. My website is working fine now.</p>

**Comment by Noorfatima777** [comment]
my website is not showing , just showing  best manager  and nothing more , https://noorfatima777.github.io/BestManager/
